### ISSUE TITLE
shaman ele 0.5 is trash, it needs more crit on the gear

### ISSUE THREAD

**Issue by kaytotes** [Project-Epoch/bugtracker#3786]
> shaman ele 0.5 is trash, it needs more crit on the gear

.go xyz -8825.060000 634.533000 95.074800 0 2.251290

**Comment by Shadownflame** [comment]
I don't think this set necessarily needs more crit. There is a total of 3% hit and 2% crit on this set, while i.e. Ret paladin set has 2% hit and 3% crit. These should be the healthy values for the buffed preraid gear (original sets only had 1-2% total at most)

There are a few other class sets which rather need to be nerfed. Like Mages get 5% hit and 2% crit (why do they get so much hit?) or Warriors get 3% hit and 4% crit (too much crit here, they also get crit from agi), or Hunters get 4% hit and 3% crit (they also get agi). While certainly some degree of randomness is a part of vanilla feel and not each peace of gear needs to be equally useful, I do find it concerning that some classes get significantly higher of these stats than others, particularly those who also benefit from Agility. These gaps are too large and favor classes which are already busted in raids.

When you consider that i.e. original Beastmaster set only had 1% crit and 1% hit, and original Five Thunders set had 2% crit, you find that shamans got the short end of the stick when Epoch decided to rebalance the sets, and an already overpowered class like Hunter received significantly higher extra stats. But I do not see the solution in bloating the stats on the shaman set to match it - rather the sets with too many stats need to be tuned down.

The only thing questionable about this ele shaman set is the Spell Penetration on the helmet. Melee sets had every redundant stat (i.e. spirit) completely removed, every single stat on their PVE gear is useful for PVE. Spell penetration isn't useful for PVE. 99% of raid bosses only have 15 spell resistance against your spells after COE, which cannot be modified by Spell Pen either, and the few which have extra specific resistance higher than what warlocks' COE can mitigate, mostly tend to be so high that a little spell pen on your gear isn't going to make a difference and you simply won't attack with the given spell school. It's basically a wasted stat, which is fine for flavor, but shouldn't take any space from a meaningful stat in the stat budget. It is especially glaring because the original Coif of the Five Thunders actually had crit, so this piece received an outright nerf instead of a lesser buff.
